widgets/S3-batchsink.json

```diff
@@ -110,7 +110,7 @@
           "widget-attributes": {
             "layout": "inline",
```

Comment:
nit: Do we support this property? Was this added newly?

Reply:
will remove

Reply:
@elfenheart @ajainarayan We have this property in radio button widget as well to display the buttons as inline or as block elements. Let me know if I have to change it there as well ( I'm assuming we use it)